Modules/core/Controller/CorespaceController.php

```diff
@@ -57,7 +57,10 @@ public function viewAction($id_space) {
 
         $modelConfig = new CoreConfig();
         $space_home_page = $modelConfig->getParamSpace('space_home_page', $id_space);
-        if ($space_home_page != "") {
+
+        $showCom = ($space_home_page == "comhome");
+
+        if ($space_home_page != "" && !$showCom) {
```

Comment:
if no space home page defined, it defaults  to space id. 


Following redirect will redirect in this case to spaceid/spaceid

Reply:
$space_home_page = $modelConfig->getParamSpace('space_home_page', $id_space)  will return $id_space if space_home_page has never been defined...

Reply:
I'm probably missing something here... 
On this PR, CoreConfig::getParamSpace():

```
/**
     * Get a parameter
     * @param string $key
     * @return string value
     */
    public function getParamSpace($key, $id_space) {
        $this->loadParams($id_space);
        if(!isset(self::$params[$id_space])) {
            return "";
        }
        if(!isset(self::$params[$id_space][$key])) {
            return "";
        }
        return self::$params[$id_space][$key];
    }
```

And, in db, the keyname "space_home_page" doesn't exist for a space if it has never been defined. In this case, getParamSpace() returns an empty string ($this->loadParams() just setting self::$params as an array [$id_space][$key]). No ?

Reply:
ah ok, my fault, I was thinking about an other option to get config params where second param is default value...

Reply:
Ah, ok :). No problem.